.github/mcp/mcp_pytest.py

```diff
@@ -45,6 +44,8 @@
 
         git checkout pr_branch
 
+        pip install --user -e .[all]
+
```

Comment:
```suggestion
        pip install --user .[all]
```
doesn't need to be editable

Reply:
done